Migration.Tool.Extensions/README.md

```diff
@@ -1,123 +1,78 @@
-## Custom migrations
-
-Samples:
-- `Migration.Tool.Extensions/CommunityMigrations/SampleTextMigration.cs` contains simplest implementation for migration of text fields
-- `Migration.Tool.Extensions/DefaultMigrations/AssetMigration.cs` contains real world migration of assets (complex example)
-
-To create custom migration:
-- create new file in `Migration.Tool.Extensions/CommunityMigrations` (directory if you need more files for single migration)
-- implement interface `Migration.Tool.KXP.Api.Services.CmsClass.IFieldMigration`
-  - implement property rank, set number bellow 100 000 - for example 5000
-  - implement method shall migrate (if method returns true, migration will be used)
-  - implement `MigrateFieldDefinition`, where objective is to mutate argument `XElement field` that represents one particular field
-  - implement `MigrateValue` where goal is to return new migrated value derived from `object? sourceValue`
-- finally register in `Migration.Tool.Extensions/ServiceCollectionExtensions.cs` as `Transient` dependency into service collection. For example `services.AddTransient<IFieldMigration, AssetMigration>()`
-
-## Custom class mappings for page types
```

Comment:
IMHO the examples for custom class mappings are helpful as well. It contains real scenario samples based on which we have implemented this extension module. Would it be possible to mention custom class mapping scenarios in the new version? Or at least keep the link to `Migration.Tool.Extensions/ClassMappings/ClassMappingSample.cs`.